cpp/src/lib/ReportGenerator.cpp

```diff
@@ -12,9 +12,14 @@
 #include "storage/Repository.h"
 #include "purchase/Invoice.h"
 #include "purchase/PurchasedBook.h"
+#include "finance/CurrencyConverter.h"
 
 using namespace purchase;
 
+static double getRoundedValueOf(double value) {
+	return (double)((int)(value * 100 + 0.5)) / 100;
```

Comment:
Prefer using C++ casts instead of C-Style casts

Reply:
You mean using 'static_cast<double>'?

Reply:
In this instance yes
In a general fashion, we no longer use C-Style casts and choose the most appropriate within the list of C++ casts
The reason for that is that when a C-Style cast in encountered by the compiler, it will apply a set a rule to decide which C++ cast to use and it can sometimes choose one you might not think/want.
https://en.cppreference.com/w/cpp/language/explicit_cast

Reply:
Ok done, i just submitted a fix with this